features/waterOrdering/index.js

```diff
@@ -65,23 +59,37 @@ const passedEnoughTimeFromLastOrder = async (robot) => {
   return lastWaterOrder === null || isEnoughTimePassed(lastWaterOrder)
 }
 
+const withRespect = ({ text }) => text.toLowerCase().includes('please')
+
+const getResponsesList = async (bot, message) => {
+  const { good, sendingError, noRespect } = replies
```

Comment:
If you decide to refactor this feature maybe it is good time to give normal names for these replies. `good` and `sendingError` sound like something strange. 🙂Thoughts?